src/Miso/Internal.hs

```diff
@@ -240,7 +255,7 @@ unmount
 unmount mountCallback app@App {..} cs@ComponentState {..} = do
   undelegator componentMount componentVTree events (logLevel `elem` [DebugEvents, DebugAll])
   freeFunction mountCallback
-  liftIO (mapM_ killThread componentSubThreads)
+  liftIO $ mapM_ killThread =<< M.elems <$> liftIO (readIORef componentSubThreads)
```

Comment:
No need to extract list of elements from the map, since Map itself is traversable
```suggestion
  liftIO $ mapM_ killThread =<< readIORef componentSubThreads
```